rasgoql/rasgoql/__init__.py

```diff
@@ -0,0 +1,43 @@
+"""
+RasgoQL package entrypoint
+"""
+from typing import Union
+
+import webbrowser
+
+from .data import DWCredentials, SnowflakeCredentials, SnowflakeDataWarehouse
+from .errors import ParameterException
+from .main import RasgoQL
+from .version import __version__
+
+__all__ = [
+    '__version__',
+    'open_docs',
+    'connect',
+]
+
+DW_MAP = {
+    'snowflake': SnowflakeDataWarehouse,
+    #'bigquery': BigQueryDataWarehouse,
+    #'postgres': PostgresDataWarehouse,
+}
+
+
+def open_docs():
+    """
+    Open Rasgo Docs page
+    """
+    url = 'https://github.com/rasgointelligence/RasgoQL'
```

Comment:
Do you wanna send them to docs.rasgoql ?

Reply:
yes i do